slowmo.owl

```diff
@@ -80,38 +122,88 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
     
 
 
-    <!-- http://purl.obolibrary.org/obo/BFO_0000050 -->
+    <!-- http://example.com/slowmo#HasCandidate -->
+
+    <owl:ObjectProperty rdf:about="http://example.com/slowmo#HasCandidate">
+        <rdfs:label>has candidate</rdfs:label>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://example.com/slowmo#IsAbout -->
 
-    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/BFO_0000050"/>
+    <owl:ObjectProperty rdf:about="http://example.com/slowmo#IsAbout">
+        <rdfs:label>is about</rdfs:label>
+    </owl:ObjectProperty>
     
 
 
-    <!-- http://purl.obolibrary.org/obo/RO_0002434 -->
+    <!-- http://example.com/slowmo#IsAboutISR -->
 
-    <owl:ObjectProperty rdf:about="http://purl.obolibrary.org/obo/RO_0002434"/>
+    <owl:ObjectProperty rdf:about="http://example.com/slowmo#IsAboutISR">
+        <rdfs:subPropertyOf rdf:resource="http://example.com/slowmo#IsAbout"/>
+        <rdfs:label>is about ISR</rdfs:label>
+    </owl:ObjectProperty>
     
 
 
-    <!-- http://purl.obolibrary.org/obo/slowmo#slowmo_0000001 -->
+    <!-- http://example.com/slowmo#IsAboutPerformer -->
```

Comment:
This is a much nicer IRI to deal with internally.  I concur with the decision to make the IRIs human readable camel case.